src/main/scala/exercises/06_Collections.scala

```diff
@@ -113,9 +99,7 @@ object Collections {
    * As always, use either build-in functions, or roll your own way via a
    * pattern match or some other method.
    */
-  def oddElements(iList: List[Int]): List[Int] = {
-    error("fix me")
-  }
+  def oddElements(iList: List[Int]): List[Int] = iList.foldRight(List(): List[Int])({(l,acc) => if (l % 2 != 0)  l::acc else acc })
```

Comment:
I chose to use `filter` for this one, which results in a slightly shorter implementation.  Why did you decide to use `foldRight` instead of `fold` or `foldLeft` though?  To me, it would be more intuitive to use `foldLeft` and append to the end of the empty list.  I know I had trouble and spent unnecessary time figuring out how to append to the end of the list. Was this also possibly your motivation for using `foldRight`?